charts/gardener-extension-kupid/templates/clusterpodschedulingpolicy.yaml

```diff
@@ -20,6 +20,10 @@ spec:
   podSelector:
     matchLabels:
 {{ toYaml $policy.podLabels | indent 6 }}
+  {{- if $policy.podExpressions }}
```

Comment:
Please add field `podExpressions` to the [`policies` sample](https://github.com/gardener/kupid/blob/8a6f56236f38fbc5e85dd7637c7704071be29712/charts/gardener-extension-kupid/values.yaml#L41-L64) in values.yaml

Reply:
Thanks addressed with 838c1e960a0562913b883e2649d9bfd8fd18b23f